app/helpers.php

```diff
@@ -1100,6 +1100,15 @@ function concat_path($paths)
     return implode('/', array_filter($paths, 'present'));
 }
 
+function platform_issue_text($issue)
+{
+    return match ($issue) {
+        'lazer' => 'laz',
```

Comment:
`lzr` feels like a better acronym than `laz`, thoughts?
```suggestion
        'lazer' => 'lzr',
```